src/services/search/service/search-index.ts

```diff
@@ -37,6 +37,10 @@ type GSearchIndex = {
   num_entries: number;
 };
 
+type IndexList = {
```

Comment:
> Do we want to export these types as well? Would make typing mirage a bit easier

For "service-sanctioned" types/schemas like this I think exporting (and matching their name, like you've done) is a good idea.

You could add a `@see https://docs.globus.org/api/search/reference/index_list/#indexlist` for documentation purposes too.